src/Tizen.NUI/src/public/BaseComponents/AnimatedImageView.cs

```diff
@@ -386,16 +386,16 @@ public void SetValues()
         [EditorBrowsable(EditorBrowsableState.Never)]
         protected override void UpdateImage()
         {
-            if(!imagePropertyUpdatedFlag) return;
+            if (!imagePropertyUpdatedFlag) return;
 
-            if (resourceURLs?.Count != 0)
+            if (resourceURLs?.Count != 0 && resourceURLs != null)
```

Comment:
could be better
(resourceURLs != null && resourceURLs.Count != 0)
as we do not check null twice.

Reply:
thank you for review.
I will fix it.

Reply:
I have fixed => https://github.com/Samsung/TizenFX/pull/4060/files#diff-7598b59f76bc7253a76f939a74012377377ad684adf2f25600603f3c7c12b3f9R391
Thank you.